mpz-core/src/block.rs

```diff
@@ -1,35 +1,35 @@
-use aes::BlockDecrypt;
-use cipher::{consts::U16, generic_array::GenericArray, BlockCipher, BlockEncrypt};
+use cipher::{consts::U16, generic_array::GenericArray};
+use clmul::Clmul;
 use core::ops::{BitAnd, BitXor};
-use itybity::{BitLength, GetBit, Lsb0, Msb0};
-use rand::{CryptoRng, Rng};
+use itybity::{BitIterable, BitLength, GetBit, Lsb0, Msb0};
+use rand::{distributions::Standard, prelude::Distribution, CryptoRng, Rng};
 use serde::{Deserialize, Serialize};
-use std::convert::{From, TryInto};
+use std::convert::From;
 
 /// A block of 128 bits
```

Comment:
But a user who instantiates `let b = Block::new(bytes)` and later calls b.lsb() must know in which order to place their `bytes`, otherwise they will get  the wrong lsb, no?

Reply:
@sinui0 ---^

Reply:
I agree with that direction. But using `color_bit` would leak the GC terminology. It can be called generically like e.g. `last_bit`or `rightmost_bit`